spec/acceptance_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'open3'
 
-describe 'Acceptance', integration: true do
+describe 'Acceptance' do
   it 'plays a draw game' do
-    stdout_str, status = Open3.capture2('ruby bin/run', stdin_data: "1\n2\n3\n5\n4\n6\n8\n7\n9\n")
+    stdout_str, status = Open3.capture2('SEED=1234 ruby bin/run', stdin_data: "1\n5\n6\n8\n3\n")
```

Comment:
Did you try setting the seed in this file and removing the following line from the script? (removing everything that it implies)
```ruby
random = Random.new(ENV['SEED'].to_i || Random.new_seed)
```